### DISCUSSION TITLE
How can I edit released environments?

### DISCUSSION THREAD

**Discussion by strykgt** [microsoft/AirSim#3417] [Support Q&A]
I tried to edit the environment received from https://github.com/microsoft/AirSim/releases using an unreal engine editor

so i pasted plugin folder from airsim/unreal/plugins into environment project and modified .uproject file

But when I ran the .uproject file, the following error occurred.

![캡처](https://user-images.githubusercontent.com/50356223/108798921-466c0b80-75d2-11eb-9907-8f83978de82c.PNG)

what should i do to solve this problem??

**Comment by ahmed-elsaharti** [comment]
Hi @GT-K, the environments in the releases section are packaged/cooked binaries. This means that they're not editable (at least as far as my understanding of the packaging process goes)
